core/src/main/java/org/bitcoinj/core/Block.java

```diff
@@ -445,13 +446,20 @@ public String toString() {
      */
     @VisibleForTesting
     public void solve() {
+        Duration fairlyLong = Duration.ofSeconds(5);
```

Comment:
A better name might be `warningThreshold`.

Reply:
Ok, I'll change it. Though sometimes I like a bit colloquial names.